### ISSUE TITLE
line-cmt doesn't stop at newline

### ISSUE THREAD

**Issue by causal-agent** [protoflex/parse-ez#6]
`line-cmt` seems to always read to EOF rather than to newline.

``` clojure
user=> (parse #(line-cmt "#") "# foo\n bar")
"#foo\n bar"
```

Passing "#" and "\n" to `blk-cmt` results in the expected behaviour (although it does not play nice with EOF)

``` clojure
user=> (parse #(blk-cmt "#" "\n") "# foo\n bar")
```

```
Exception Parse Error: Unexpected input: "b"; Expecting "
" at line 2, col 2  protoflex.parse/throw-ex (parse.clj:274)
```

**Comment by protoflex** [comment]
Sorry for the delay in responding.  Here is the new line-cmt function to be included in next release:

``` clojure
defn line-cmt
  "Reads and returns a line comment as specified by the begin marker.
  Throws an exception if the specified block-comment doesn't occur at the
  current position."
  [beg]
  (if beg 
    (->> (with-trim-off (series #(string beg) #(regex #"[^\n]*\n?"))) 
         (apply str))))
```

**Comment by protoflex** [comment]
Fixed in 0.4.1.
